CONTRIBUTING.md

```diff
@@ -251,6 +254,15 @@ For information on how to add a service to our f5-config.py script, please see h
 #### Adding MaaS (Monitoring) plugins for new services<a name="add-maas-plugins"></a>
 For information on how to create MaaS plugins, please see: https://github.com/rcbops/rpc-openstack/blob/master/maas/plugins/README.md
 
+#### Integration tests<a name="integration-tests"></a>
+The newly introduced OpenStack project must have some form of integration tests. Whether that be through [tempest](http://docs.openstack.org/developer/tempest/) or our [jenkins-rpc](https://github.com/rcbops/jenkins-rpc/blob/master/README.md) gating jobs. Most OpenStack services already come with some scenario tests as part of the tempest test suite. However, if the newly introduced service does not, then integration tests can be added to the jenkins-rpc CI/CD pipeline to run on a per-commit basis.
```

Comment:
Most OpenStack services provide scenario tests as part of the tempest test suite. If not, add integration tests to the `jenkins-rpc` CI/CD pipeline that run on each commit.